apps/datahub-e2e/src/support/commands.ts

```diff
@@ -128,3 +130,41 @@ Cypress.Commands.add('clearFavorites', () => {
       })
     })
 })
+
+Cypress.Commands.add('interceptDataset', (id) => {
+  cy.fixture(`datasets/${id}.json`).then((fixtureData) => {
+    cy.intercept(
+      'POST',
+      `/geonetwork/srv/api/search/records/_search?bucket=bucket`,
+      (req) => {
+        if (
+          req.body.query?.ids?.values.includes(
+            '775b7660-b03b-443a-9817-be82ecd0ef07'
```

Comment:
Just for my understanding, why is this condition actually needed, as the command is currently only called for this dataset?

Reply:
Oops I forggot to make it generic, I updated to: 
```
if (req.body.query?.ids?.values.includes(id))
```
https://github.com/camptocamp/mel-dataplatform/pull/118/files#diff-dc85986d4f606f14c248534d03723c0e6d7af1b67c90d73565774bfa2d2dc445R140

The condition is needed because there is no other way to identify the request dedicated to the dataset info since `/geonetwork/srv/api/search/records/_search?bucket=bucket` is also used to populate filters, queryscore, etc...

if there was a route like `/geonetwork/srv/api/search/records/dataset/775b7660-b03b-443a-9817-be82ecd0ef07` we wouldnt need to analyse the request payload, a simple intercept cy.intercept('POST',
      `/geonetwork/srv/api/search/records/dataset/775b7660-b03b-443a-9817-be82ecd0ef07` ) would have been sufficient.